starkinfra/issuingtoken/__issuingtoken.py

```diff
@@ -0,0 +1,221 @@
+from json import dumps
+from starkinfra.utils import rest
+from starkcore.utils.api import api_json
+from ..utils.parse import parse_and_verify
+from starkcore.utils.resource import Resource
+from starkcore.utils.checks import check_datetime, check_date
+
+
+
+class IssuingToken(Resource):
+    """# IssuingToken object
+    The IssuingToken object displays the information of the tokens created in your Workspace.
+    ## Attributes (return-only):
+    - card_id [string]: unique id returned when IssuingCard is created. ex: "5656565656565656"
```

Comment:
Correto, mas essa não é descrição que precisa ter nessa docstring
`card ID which the token is bounded to`